Applications/Spire/Source/Ui/DropDownList.cpp

```diff
@@ -61,18 +32,47 @@ DropDownList::DropDownList(ListView& list_view, QWidget* parent)
   m_panel->set_closed_on_blur(true);
   m_panel->layout()->itemAt(0)->widget()->setSizePolicy(QSizePolicy::Minimum,
     QSizePolicy::Preferred);
+  if(auto list_item = m_list_view->get_list_item(0)) {
+    setMaximumHeight(10 * list_item->sizeHint().height());
```

Comment:
Can size hint be used for this? The issue with stuff like this is that it inhibits future composition. No future component will be able to override this property without "fighting" with the drop down list.